src/vapid-helper.js

```diff
@@ -86,7 +86,7 @@ function validateSubject(subject) {
         + 'sending notifications.');
     }
   } catch (err) {
-    throw new Error('Vapid subject is not a valid URL. ' + subject);
+    throw new Error('Vapid subject is not a valid URL. ' + subject, { cause: err });
   }
```

Comment:
What if we moved the try-catch to only wrap the `const subjectParseResult = new URL(subject);` line?

Reply:
I think it would work as well since it probably was the main reason the try-catch was put there in the first place. And it will also make the error on the console more clean/targetable.

I added the cause solution trying to change as minimun as possible

Reply:
I'd prefer moving the try-catch, it means a bit more changes, but it seems nicer to avoid throwing an exception only to catch it and rethrow another exception with a different message.